docs/content/get-started/quickstarts/spark-client-mode.md

```diff
@@ -0,0 +1,46 @@
+# Spark Quickstart
+
+In this document, we will show you the steps to submit a simple FeatHub job to a standalone Spark cluster. 
```

Comment:
Let's wrap the contents of this document so that each line contains no longer than 80 characters.